server/routes/routes.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express')
+const router = express.Router()
+const urllib = require('urllib')
+
+const teamToIDs = require('../../DB/teams')
+const dreamTeam = require('../../DB/dreamTeam')
+
+router.get('/teams/:teamName', (req, res) => {
+    const { teamName } = req.params
+
+    urllib.request('http://data.nba.net/10s/prod/v1/2018/players.json', (err, data) => {
+        const players = JSON.parse(data.toString()).league.standard
+
+        const filteredPlayers = players.filter((p) => {
+            if (p.teams.length) {
+                return p.teams[p.teams.length - 1].teamId == teamToIDs[teamName] && p.isActive
+            } return false
+        })
+            .map((p) => ({
+                firstName: `${p.firstName}`,
```

Comment:
you could just write -->>. firstName: p.firstName
no need for the `${XXX}` here

that goes for the others also